### ISSUE TITLE
Loss value too big while training the custom model

### ISSUE THREAD

**Issue by saqibshakeel035** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#408]
I am following [this tutorial](https://github.com/EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10)  for the training of my data (Train images 4 and test images 2) all are car images. Everything was running fine starting from loss value =3 but as the training process proceeded, I am getting very big values. My question here is to know the reason behind this weird behavior and how can I possibly train my model right???

`I1123 23:14:24.024599  8896 learning.py:507] global step 5746: loss = 3332910014858788864000.0000 (8.303 sec/step)
INFO:tensorflow:global step 5747: loss = 3994775060170829463552.0000 (8.519 sec/step)
I1123 23:14:32.546626  8896 learning.py:507] global step 5747: loss = 3994775060170829463552.0000 (8.519 sec/step)
INFO:tensorflow:Recording summary at step 5747.
I1123 23:14:38.774678  6928 supervisor.py:1050] Recording summary at step 5747.
INFO:tensorflow:global step 5748: loss = 1396395248399239086080.0000 (10.488 sec/step)
I1123 23:14:43.037031  8896 learning.py:507] global step 5748: loss = 1396395248399239086080.0000 (10.488 sec/step)
INFO:tensorflow:global step 5749: loss = 892919831656981856256.0000 (8.293 sec/step)
I1123 23:14:51.332884  8896 learning.py:507] global step 5749: loss = 892919831656981856256.0000 (8.293 sec/step)
INFO:tensorflow:global step 5750: loss = 893498403471610609664.0000 (8.364 sec/step)
I1123 23:14:59.700745  8896 learning.py:507] global step 5750: loss = 893498403471610609664.0000 (8.364 sec/step)
INFO:tensorflow:global step 5751: loss = 4019180066551550181376.0000 (8.456 sec/step)
I1123 23:15:08.158498  8896 learning.py:507] global step 5751: loss = 4019180066551550181376.0000 (8.456 sec/step)
INFO:tensorflow:global step 5752: loss = 4025371108664301060096.0000 (8.323 sec/step)
I1123 23:15:16.482330  8896 learning.py:507] global step 5752: loss = 4025371108664301060096.0000 (8.323 sec/step)
INFO:tensorflow:global step 5753: loss = 9825390717043526860800.0000 (8.322 sec/step)
I1123 23:15:24.806868  8896 learning.py:507] global step 5753: loss = 9825390717043526860800.0000 (8.322 sec/step)
INFO:tensorflow:global step 5754: loss = 9830483162322176049152.0000 (8.451 sec/step)
I1123 23:15:33.259524  8896 learning.py:507] global step 5754: loss = 9830483162322176049152.0000 (8.451 sec/step)
INFO:tensorflow:global step 5755: loss = 2754848514362811875328.0000 (8.394 sec/step)
I1123 23:15:41.656417  8896 learning.py:507] global step 5755: loss = 2754848514362811875328.0000 (8.394 sec/step)
INFO:tensorflow:global step 5756: loss = 9176984960692859699200.0000 (8.338 sec/step)
I1123 23:15:49.997432  8896 learning.py:507] global step 5756: loss = 9176984960692859699200.0000 (8.338 sec/step)
INFO:tensorflow:global step 5757: loss = 9177446579654665175040.0000 (8.335 sec/step)
I1123 23:15:58.334255  8896 learning.py:507] global step 5757: loss = 9177446579654665175040.0000 (8.335 sec/step)`

**Comment by saqibshakeel035** [comment]
While debugging the training process, I cam to know that the content of **labelmap.pbtxt** was also present at the top of  **.config**  file which obviously was my mistake while modifying the config file. Once I removed the those lines of codes that I pasted by mistake from labelmap file and retrained my model. I successfully trained my desired model.
